Cryptographic columnar cipher implemented in Java

Parshwa52 opened the pull request:
I have created a cryptographic columnar cipher used in networking transmission.

dubesar reviewed (changes requested):
Hey please see the review and also again add in some brief comment regarding the concept.

Parshwa52 commented:
I have added identation in Producer Consumer problem along with comments 
and also in columnar cipher I have added link for proper explanation and comments added with indentation
